src/cockpit/389-console/src/lib/database/globalPwp.jsx

```diff
@@ -1117,300 +1146,302 @@ export class GlobalPwPolicy extends React.Component {
+                                isLoading={this.state.saving}
+                                spinnerAriaValueText={this.state.saving ? "Saving" : undefined}
+                                {...extraPrimaryProps}
+                            >
+                                {saveBtnName}
+                            </Button>
+                        </Tab>
+                        <Tab eventKey={3} title={<TabTitleText><b>Syntax Checking</b></TabTitleText>}>
+                            <Form className="ds-margin-top-xlg ds-margin-left" isHorizontal>
+                                <Grid title="Enable password syntax checking (passwordCheckSyntax).">
+                                    <GridItem span={12}>
+                                        <Checkbox
+                                            id="passwordchecksyntax"
+                                            isChecked={this.state.passwordchecksyntax}
+                                            onChange={(checked, e) => {
+                                                this.handleSyntaxChange(e);
+                                            }}
+                                            label="Enable Password Syntax Checking"
+                                        />
+                                    </GridItem>
+                                </Grid>
+                                {pwSyntaxRows}
+                            </Form>
+                            <Button
+                                isDisabled={this.state.saveSyntaxDisabled}
+                                variant="primary"
+                                className="ds-margin-top-xlg ds-margin-left"
+                                onClick={this.saveSyntax}
+                                isLoading={this.state.saving}
+                                spinnerAriaValueText={this.state.saving ? "Saving" : undefined}
+                                {...extraPrimaryProps}
+                            >
+                                {saveBtnName}
+                            </Button>
+                        </Tab>
+                    </Tabs>
                 </div>;
         }
 
         return (
-            <div>
-                <Row>
-                    <Col sm={5}>
-                        <ControlLabel className="ds-suffix-header ds-margin-top-lg ds-margin-left-sm">
-                            Global Password Policy
-                            <Icon className="ds-left-margin ds-refresh"
-                                type="fa" name="refresh" title="Refresh global password policy settings"
-                                onClick={this.loadGlobal}
-                            />
-                        </ControlLabel>
-                    </Col>
-                </Row>
+            <div className={this.state.saving ? "ds-disabled" : ""}>
+                <Grid>
+                    <GridItem span={3}>
+                        <h4>Global Password Policy <FontAwesomeIcon
```

Comment:
By some reason, the refresh button is out of the way (may be the grid is too small).
![image](https://user-images.githubusercontent.com/5634892/124970136-d3ff4380-e027-11eb-8f1a-f1ff3d30d6fd.png)

Reply:
Fixed...